smokefree/sites/all/themes/smokefreeteen/templates/footer.tpl.php

```diff
@@ -127,7 +127,7 @@ class="logobar links"
       <input type="hidden" name="filter"          value="0" />
       <input type="hidden" name="getfields"       value="*" />
       <label for="searchbox-footer"
-             class="hiddenSearch hidden">Search</label>
+             class="hiddenSearch sr-only">Search</label>
```

Comment:
Why are we making these changes? I can see them throughout the review. Are they required?

Reply:
Yes, they were required by Chris Deloose.